backend/internal/storage/postgres/schema/user.go

```diff
@@ -328,6 +328,103 @@ func (r *UserRepository) GetUserFeed(ctx context.Context, id uuid.UUID) ([]*mode
 
 }
 
+func (r *UserRepository) CreateSection(ctx context.Context, sectiontype models.SectionType) (models.SectionType, error) {
```

Comment:
just a question to understand the system: is the section_type table just gonna be populated once and have the 5 sections outlined by design? or will we need to call this endpoint and add others in the future